src/app/components/header/find-movie.component.ts

```diff
@@ -10,12 +11,13 @@ export class FindMovieComponent implements OnInit {
   public movieTitle: string = "";
 
 
-  constructor(private findService : FindMovieService) { }
+  constructor(private findService : FindMovieService, private rout : Router) { }
```

Comment:
it's not clear why you keep making word contractions like "rout", for me it's always good to have easy to read full short names when you get to your code after some break. it's okay to leave as is, up to you